AlphaWallet/Core/CoinTicker/CoinTickersFetcher.swift

```diff
@@ -277,8 +277,26 @@ fileprivate struct Ticker: Codable {
         //   }
         //},
         //This means we can only match solely by symbol, ignoring platform matches. But this means it's easy to match the wrong ticker (by symbol only). Hence, we at least remove those chains we don't think are supported
+        //NOTE maybe its need to handle values like: `"0x270DE58F54649608D316fAa795a9941b355A2Bd0/token-transfers"`
+
         guard isServerSupported(tokenObject.server) else { return false }
-        if let (_, contract) = platforms.first(where: { platformMatches($0.key, server: tokenObject.server) }) {
+        if let (_, maybeContractValue) = platforms.first(where: { platformMatches($0.key, server: tokenObject.server) }) {
+            func maybeAddressValue(from str: String) -> AlphaWallet.Address? {
+                let rawValue = str.trimmed
+                if rawValue.isEmpty {
+                    //CoinGecko returns nullAddress as the value (contract) in `platforms` for tokens is sometimes an empty string: `"platforms" : { "ethereum" : "" }`, so we use the 0x0..0 address to represent them
+                    return Constants.nullAddress
+                } else if let value = AlphaWallet.Address(string: rawValue) {
```

Comment:
Why is it faster if we delay `AlphaWallet.Address(string:)` till we do it here? Wouldn't we potentially be doing it more often for the same `rawValue`?

Reply:
its faster as we filter out a lot of objects when do:
`platformMatches($0.key, server: tokenObject.server)` and we don't create not needed addresses